### ISSUE TITLE
Hangs on fetching UUID

### ISSUE THREAD

**Issue by afestein** [Shirk/motion-crescentia#5]
Thanks for the work on this as I've been struggling with other outdated gems for frank & calabash. 

However, I'm getting `rake crescentia:run 'iPhone 6s (9.0) [22451E75-34E0-46B4-A5DC-2B92FF8658A3]'`, hanging with:

`Run Fetching the UUID for 'iPhone 6s (9.0) [22451E75-34E0-46B4-A5DC-2B92FF8658A3]'`

I'm going to download and test on an 8.4 Simulator but there seems to be an issue here.

**Comment by afestein** [comment]
Same result fetching UUID for 'iPhone 4s (8.4)'. I am using XCode 7, perhaps that is the issue?

**Comment by HealsCodes** [comment]
Hey Adrian, 
thanks for the feedback. 

I'll look into the problem once I find some time to spare (probably on the weekend).

The QUID fetcher isn't overly complex so I'm hoping for an easy fix. 
Also it looks like a good time to bump to the latest cucumber server libs.

**Comment by afestein** [comment]
Great :smile:

**Comment by afestein** [comment]
Hiya @Shirk. Did you get an opportunity to investigate this?

**Comment by HealsCodes** [comment]
Yes indeed I did - and after some head scratching I looked at your original report.
Could you please try without having the UUID in the device name?

`rake crescentia:run 'iPhone 6s (9.0)'`

I assume your comment with the 8.4 also had an UUID in the original call?

**Comment by afestein** [comment]
Thanks for the reply. Still hangs whether or not I specify the UUID though. 

![screen shot 2015-10-28 at 9 58 21 am](https://cloud.githubusercontent.com/assets/6790096/10776144/88929b62-7d5a-11e5-8d68-911d262cdd2b.png)

**Comment by gzfrancisco** [comment]
Same here. It could be related to the same command without target.

When I run the same command without target it hangs out and never shows the list.

**Comment by gzfrancisco** [comment]
The problem is this line:
https://github.com/Shirk/motion-crescentia/blob/master/lib/motion/project/crescentia.rb#L54

It hangs out when the call include the grep method.

How do you need to have the array?, maybe I could change it.

**Comment by gzfrancisco** [comment]
It's weird because I opened the irb and executed that method and it hangs twice, but then I tried third and runs ok.

then run the rake task and all worked fine with and without the target.
